components/customcomponents/CustomAddToCart.vue

```diff
@@ -0,0 +1,105 @@
+<template>
```

Comment:
rename CustomAddToCart.vue to KiboAddToCart.vue

Reply:
Renamed & Moved to components directory